screens/changePassword/ChangePasswordScreen.tsx

```diff
@@ -27,113 +31,117 @@ type FormSubmission =
   | { status: "valid"; submit: () => void }
   | { status: "invalid"; errors: string[] }
 
-const beginChangePassword = () => {}
-
 export const ChangePasswordScreen = ({
   onPasswordChangeSubmitted = changePassword
 }: ChangePasswordProps) => {
-  const [currentPassword, setCurrentPassword] = useState("")
-  const [newPassword, setNewPassword] = useState("")
-  const [reEnteredPassword, setReEnteredPassword] = useState("")
-  const [isValidForm, setIsValidForm] = useState(false)
+  const [currentPassword, setCurrentPassword] = useState<string>("")
+  const [newPassword, setNewPassword] = useState<string>("")
+  const [reEnteredPassword, setReEnteredPassword] = useState<string>("")
+
+  const validateForm = () => {
+    const passwordRegex: RegExp =
+      /^(?=.*[A-Z])(?=.*[!@#$%^&*])(?=.*[0-9]).{8,}$/
+    const formValid: boolean =
```

Comment:
Can take this into the body of the component (can use this to tell if the form is valid)